manifests/init.pp

```diff
@@ -226,13 +226,12 @@
   }
 
   file { 'sources.list':
-    ensure  => file,
-    path    => $::apt::sources_list,
-    owner   => root,
-    group   => root,
-    mode    => '0644',
-    content => $sources_list_content,
```

Comment:
Do we want to keep the content? Otherwise it can contain sources manually maintained. I think it's valid to pass content with `ensure => absent`.

Reply:
the current behavior is the file content is untouched if the file is not "purged", so it's consistent with the current behavior. I think keeping sources manually maintained was exactly the point of this...?